lib/services/sf.js

```diff
@@ -1481,7 +1481,11 @@ StateRenewing.prototype.enter = function (context)
       json:
         {
           'REQUEST_TYPE': 'RENEW',
-          'oldSessionToken': this.tokenInfo.getSessionToken()
+          'oldSessionToken': this.tokenInfo.getSessionToken(),
+          data: {
+            CLIENT_APP_ID: this.connectionConfig.getClientType(),
```

Comment:
In other places client info data is added to object json.data so it is set as a request body too. As a header it is added only for 'login requests' category. In my opinion it is better to set this in body and copy to headers when it is needed. I'm not sure this logic because I didn't find the decision about headers, still looking for.